CompetitionTemplate/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -53,12 +54,15 @@ public void setupDriveCommands(
             CalibrateHeadingCommand calibrateHeading,
             HeadingDriveCommand headingDrive,
             DriveToWallCommand driveToWall,
-            ResetRobotPositionCommand resetPosition
+            ResetRobotPositionCommand resetPosition,
+            ChevalCommandGroup cheval
             )
     {
         operatorInterface.leftButtons.getifAvailable(2).whenPressed(calibrateHeading);
         
         resetPosition.includeOnSmartDashboard("Reset Position");
+        
+        operatorInterface.leftButtons.getifAvailable(3).whileHeld(cheval);
```

Comment:
Nice idea! I created a ChevalCommandThatNeverStops for the OI to use, and the other one can be used during auto.